### DISCUSSION TITLE
KeyError: 'global_step' while installing Anything V3.0

### DISCUSSION THREAD

**Discussion by technomaniak** [azuritecoin/OnnxDiffusersUI#56] [Q&A] [answered]
I was trying to get Anything-V3.0 to work. 
I ran `python convert_original_stable_diffusion_to_diffusers.py --checkpoint_path="Anything-V3.0.ckpt" --dump_path="Anything-V3.0_diffusers"`

It gave me this error instantly:
`Traceback (most recent call last):
  File "C:\Users\'censored'\Documents\SUPER SD .1\convert_original_stable_diffusion_to_diffusers.py", line 799, in <module>
    global_step = checkpoint["global_step"]
KeyError: 'global_step'`

I use [Anything-V3.0.ckpt](https://huggingface.co/Linaqruf/anything-v3.0/tree/main) file from the huggingface site.

**Comment by azuritecoin** [comment, marked as answer]
Since it's hosted on the huggingface website and it's already in diffusers model, you can use the second conversion script directly: `convert_stable_diffusion_checkpoint_to_onnx.py --model_path="Linaqruf/anything-v3.0" --output_path="anything-v3.0_onnx"`

The script will download and convert in one step so you don't have to do the two step conversion with the `.ckpt` file.

**Reply by caprexy** [reply]
I updated my script and it worked since they added some error handling for global_step
https://github.com/huggingface/diffusers/blob/main/scripts/convert_original_stable_diffusion_to_diffusers.py
